plugins/source/github/resources/services/organizations/alerts.go

```diff
@@ -20,13 +20,19 @@ func alerts() *schema.Table {
 
 func fetchAlerts(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
 	c := meta.(*client.Client)
+	opts := &github.ListAlertsOptions{ListCursorOptions: github.ListCursorOptions{PerPage: 99}}
```

Comment:
Is there a reason to chose 99? Looks like the API supports 100:
https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28#list-dependabot-alerts-for-an-organization

![image](https://user-images.githubusercontent.com/26760571/227990254-88c35db8-d93d-403c-9350-509a023bc56f.png)

Same for the other table

Reply:
Yeah, let's do 100 I don't think we need to be concerned with hitting the limit as 100 is supported